BUILD.md

```diff
@@ -236,6 +236,8 @@ Building on Linux requires the installation of the following packages:
 
 #### Ubuntu
 
+GFXReconstruct is built and tested for SDK development using Ubuntu 20.04.  Older Ubuntu releases are unsupported but may work.
```

Comment:
" but may work" implies "may not work", which makes these words carry zero actionable information, and suggests deletion of them.

Reply:
Yes!